hack/ocp-install-nightly-art-operators.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# This script is an automation of steps 1-7 from
+# https://hackmd.io/0vbbJdZ7RXem69vCWI61ug to install the nightly ART bundle
+# build of the OpenShift operators.  The script requires the user to have a
+# employee-token created at least once before
+# (https://source.redhat.com/groups/public/teamnado/wiki/brew_registry).
+
+set -e
+
+function wait_for_all_nodes_ready() {
+  i=20
+  while [ $i -gt 0 ]; do
+    if [[ $(oc get no | grep SchedulingDisabled) ]]; then
+      # reset timer
+      i=20
+      echo "found node with SchedulingDisabled. Resetting timer..."
+    else
+      echo "Waiting ${i} times to ensure all nodes are ready..."
+      i=$((i - 1))
+    fi
+
+    sleep 1
+  done
+}	
+
+brew_credentials=$(curl --negotiate -u : https://employee-token-manager.registry.redhat.com/v1/tokens -s) #make sure you created a token before (https://source.redhat.com/groups/public/teamnado/wiki/brew_registry)
+brew_username=$(echo $brew_credentials | jq -r ".[0].credentials.username")
+brew_password=$(echo $brew_credentials | jq -r ".[0].credentials.password")
+
+cluster_version=$(oc get clusterversion version -o jsonpath='{.status.desired.version}' | cut -d '.' -f 1,2)
```

Comment:
for all command being used inside "$( )" I would check if it fails or not. If fails, shows a message and exit 1

Reply:
This script is set -e. If anything fails it will exit immediately.

We might want to add -o pipefail to make sure if a command in a pipeline fails we will catch that too, but we don't need an explicit return value check for every command.

Reply:
As Ben said, I use `set -e` at the beginning. Anyhow adding the pipefail too, makes sense.